### PR TITLE
Feature/high level isc

### LINKED ISSUE #124: Create a zeta for python desktop
This lib enables Python scripts to interact with embedded Zeta services. With this, we will have a Zeta for Python applications and this Zeta Desktop. Python scripts will work as a model for embedded software services providing good oracle for tests. 

Develop a mechanism that enables Python script communicate with Zeta Services through Zeta Forwarder for sending and receiving messages to Zeta services running on a device. It will improve radically testing for embedded software using Zeta. Zeromq is an option to implement something similar to the Zeta ISC for Desktop. 

[1] - https://learning-0mq-with-pyzmq.readthedocs.io/en/latest/pyzmq/patterns/pubsub.html

### PR DESCRIPTION
**Pull request by rodrigopex**

### CODE REVIEW

File: samples/sunlight_model/src/isc.c [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+                        if (err) {
+                            printk("publishing status %d\n", err);
+                        }
+                        // printk("Size = %u, level = %d\n", s->sunlight_level_msg.size,
+                        //        s->sunlight_level_msg.value.level);
+                        // printk("write or update\n");
+                        // for (int i = 0; i < pkt.message_info.size; ++i) {
+                        //     printk("%02X ", pkt.message[i]);
+                        // }
+                        // printk("\n");
```
Comment (lucaspeixotot) [inline samples/sunlight_model/src/isc.c]: If this code is not necessary it can be deleted.

File: samples/sunlight_model/src/isc.c [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+                    }
+                }
+                }
+                // zt_isc_net_pkt_send(&pkt);
```
Comment (lucaspeixotot) [inline samples/sunlight_model/src/isc.c]: The same here.

File: samples/sunlight_model/src/isc.c (hunk 1 above)
Comment (rodrigopex) [reply samples/sunlight_model/src/isc.c]: The whole file is not being used in fact. It was the initial ISC's implementation. Removed from the base.

File: samples/sunlight_model/src/isc.c (hunk 2 above)
Comment (rodrigopex) [reply samples/sunlight_model/src/isc.c]: The whole file is not being used in fact. It was the initial ISC's implementation. Removed from the base.

**Review by lucaspeixotot — APPROVED** [review]

**Pull request merged**